report/wine_classification_report.ipynb

```diff
@@ -225,28 +267,134 @@
-    "In addition, diversifying our evaluation metrics can provide a more comprehensive understanding of our model's performance. Metrics such as precision, recall, F1-score are good choices for imblanced class. According to our baseline model, the accuracy is of 40.33%, indicates that the most prevalent class occurs at a rate of 40%. This suggests a class imbalance (as we have three classes), prompting a closer examination of class distribution during EDA. "
+    "Figure 1: Density plots per class of wine for the 13 physiochemical properties included in the dataset."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#![Accuracy scores for training and validation sets during hyperparameter optimization](../results/figures/wine_cv_C.png)"
+    "### Preprocessing"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "```\n",
-    "{glue:figure}\n",
-    ":figwidth: 400px\n",
-    ":name: \"test_scores_df\"\n",
+    "All the features in the dataset are numerical and take on different ranges. We standardized them using  sklearn's `StandardScaler` with default parameters which does so by removing the mean and scaling to unit variance. Standardization is essential since it prevents features with large variances from dominating the objective function. StandardScaler ensures a consistent scale across features, which aligns with the assumption that logistic regression makes i.e. that features are approximately normally distributed with 0 mean and unit variance. \n",
     "\n",
-    "Accuracy and F1 scores to evaluate model performance on test data\n",
-    "```\n"
+    "We used sklearn's `ColumnTransformer` to create our preprocesser object that implemented the `StandardScaler`.  The preprocessor was fit on the train set and then used to transform both the train and test sets. \n",
+    "\n",
+    "The preprocessor object was pickled and can be found in the project directory under results/models `preprocessor_model.pickle`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Model Building"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we built our LR model with sklearn's `LogisticRegression`. Logistic regression was our linear model of choice because it is simple, easy to implement/interpret and can also extend to multiple classes (as is the case with our dataset). "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To find the optimal value of the hyperparameter `C` for our LR model that maximized the accuracy score, we performed a 10-fold cross-validated grid search with sklearn's `GridSearchCV`.\n",
```

Comment:
Minor: We could explain we used GridSearch over RandomSearch since there's only one parameter to Optimize